image_processor/inc/convolution_tools.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CONVOLUTION_TOOLS
+#define CONVOLUTION_TOOLS
+#include "standard_lib_includer.h"

```

Comment:
I don't think that such an uber-include file is common practice. I recommend including only necessary headers.